### PR TITLE
Nomysql for bulk

### PR DESCRIPTION
**Pull request by jlark**
### updates
- As influxDB gets more mature, it's good to slowly move away from storing time series data in MYSQL. As part of this, I'm making MYSQL inserts for mysql for bulks optional.
- Updating a couple field names as the schema was modified for `RMC_Status` table
- Sometimes compressed data comes in corrupted.  Adding exception handling so this doesn't create a rollbar alert every single time.
### TODO
- Figure out why compressed data is coming in corrupted as often as it is.

### CODE REVIEW

**Comment by jlark** [comment]
Hey @bumi any chance to look at this?

**Comment by bumi** [comment]
great to hear that influxdb is working well. 

why do you want to disable it for the bulk insert but not for the simple insert endpoints?
sounds to me like some data might be in MySQL but some might not and thus this data seems kinda useless?

**Comment by bumi** [comment]
what exactly is the de-compression error? and do you have an example invalid payload?

File: api.py [hunk 1]
```diff
@@ -179,7 +186,8 @@ def bulk():
         logging.debug("inserting %s into table %s"%(inserts,table))
 
         # We need to send the data to the correct table according to the type of data it is
-        insert_data(inserts,table)
+        print "got %s"% app.config['BULK_MYSQL_INSERT']
```
Comment (bumi) [inline api.py]: do we need that print? I think we should use logging.info here with a more descriptive message.

Comment (jlark) [inline api.py]: no need for the print will remove.

**Comment by jlark** [comment]
You're right, I neglected the single data endpoints will do this for them as well.

Error I recieve for for zlib is `zlib.error: Error -5 while decompressing data: incomplete or truncated stream`

**Comment by jlark** [comment]
done

**Comment by bumi** [comment]
LGTM. the name `BULK_MYSQL_INSERT` now might be a bit missleading as it is completely turning of MySQl

**Comment by bumi** [comment]
btw. when we disable MySQL now for us... do we still need the mysql code? :)

**Comment by jlark** [comment]
good point, as it's now turning off all mysql. I'll change this too.
I'll look to capture a packet with zlib failing and and open an issue for it and at it there.

**Comment by jlark** [comment]
It's happening after a fresh deploy. 
Perhaps wrong version of react?
You can see it here.
http://prod1.westeurope.cloudapp.azure.com

**Comment by bumi** [comment]
@alptilev is this comment for another issue?

**Comment by jlark** [comment]
my bad wrong thread indeed @bumi

**Pull request merged**